Lib/test/test_codecs.py

```diff
@@ -3440,5 +3440,21 @@ def search_function(encoding):
         self.assertEqual(NOT_FOUND, codecs.lookup('a\xe9\u20ac-8'))
 
 
+class EncodingNormalizationTest(unittest.TestCase):
```

Comment:
I suggest to reuse CodecNameNormalizationTest and rename CodecNameNormalizationTest.test_normalized_encoding to test_codecs_lookup().

The new test can be renamed to test_encodings_normalize_encoding().